pkgs/applications/audio/picard/default.nix

```diff
@@ -58,6 +58,8 @@ pythonPackages.buildPythonApplication rec {
     pyyaml
   ];
 
+  setupPyGlobalFlags = [ "build" "--disable-autoupdate" ];
```

Comment:
I’m unfamiliar with Python packaging and not confident about adding `build` here. Is there a more correct way to configure this?

Reply:
Looked into it. It expands into `python setup.py build --disable-autoupdate bdist_wheel`, so it's fine.